test/container_test.py

```diff
@@ -787,19 +790,21 @@ def test_cls_generator(servicer):
 
 @skip_github_non_linux
 def test_checkpointing_cls_function(servicer):
-    ret = _run_container(
-        servicer,
-        "test.supports.functions",
-        "SnapshottingCls.*",
-        inputs=_get_inputs((("D",), {}), method_name="f"),
-        is_checkpointing_function=True,
-        is_class=True,
-    )
-    assert any(isinstance(request, api_pb2.ContainerCheckpointRequest) for request in servicer.requests)
-    for request in servicer.requests:
-        if isinstance(request, api_pb2.ContainerCheckpointRequest):
-            assert request.checkpoint_id
-    assert _unwrap_scalar(ret) == "ABCD"
+    # Monkey-patched to prevent side effects with other existing connections.
+    with mock.patch("modal._container_io_manager.get_open_connections", lambda: []):
```

Comment:
Would be good to get a solution that doesn't involve patching every test.

Reply:
Perhaps a test fixture for a pre-patched io manager?